interface/themes/oe-common/acl-common.scss

```diff
@@ -0,0 +1,28 @@
+/*-------------------openemr ACL-------------------*/
+
+.loading {
+    text-decoration: blink;
+    font-size: 10px;
+    color: red;
+    font-weight: bold;
+}
+.alert {
+    font-size: 10px;
+    color: red;
+    font-weight: bold;
+}
+.section {
+    border: solid;
+    border-width: 1px;
+    margin: 10px 0 10px 0;
+    padding: 5px;
+}
+#acl select[multiple], select[size] {
+    height: auto !important;
+    width: 400px;
+}
+.section a, .section a:visited, .section a:hover {
+    text-decoration: none;
+    color: #000000 !important;
+}
+/*-------------------openemr ACL-------------------*/
```

Comment:
Is this currently in the acl page in the codebase?
If so, can it be removed from that page?
Also, the loading/alert/section stuff has really generic names. Won't it likely conflict with other stuff with same names?(since I think this ends up on a big style sheet, correct?)

Reply:
hi @zbig01 ,
Just checking if you have a response to the post above. thanks, -brady